### PR TITLE
Add event to manipulate containers

### PR DESCRIPTION
**Pull request by timkelty**
I needed a place where I could manipulate the meta container before output. Here's an example usage:

```
        Event::on(
            \nystudio107\seomatic\models\MetaLinkContainer::class,
            \nystudio107\seomatic\models\MetaLinkContainer::EVENT_INCLUDE_CONTAINER,
            function(\nystudio107\seomatic\events\IncludeContainerEvent $event) {
                $container = $event->sender;
                $include = $event->include;
                $matched = \nystudio107\seomatic\Seomatic::$matchedElement;

                $alternateMeta = $container->data['alternate'];

                $alternateMeta->hreflang[] = 'en-cn';
                $alternateMeta->href[] = $matched->url;

                $alternateMeta->hreflang[] = 'en-hk';
                $alternateMeta->href[] = $matched->url;
            }
        );
```

In this case, I needed to add some alternate hreflang tags that weren't actually part of my Craft Sites config.

For my uses, it wasn't particularly important where the event was fired, as long as it happened before any rendering. I put it in `prepForInclusion` as it seems to always get called once per container. There may be a more logical place for it, but all I really needed was to be able to manipulate the container (via `$event->sender`).

Note: I was originally going to trigger it within `normalizeContainerData`, but the data from dynamic meta didn't seem to have been added yet at that point, which I needed.

Basically, wherever the "last stop for container manipulation before rendering" is where I'm looking to hook in.

### CODE REVIEW

**Comment by khalwat** [comment]
Looking good!
